### PR TITLE
Added Zoom h2 section

### PR DESCRIPTION
**Pull request by mikehoffms**
**Rendered section for review:**
https://review.docs.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/overview?branch=pr-en-us-1848#zoom-devtools-in-or-out

### CODE REVIEW

File: microsoft-edge/devtools-guide-chromium/overview.md [hunk 1]
```diff
@@ -155,6 +155,30 @@ See [Keyboard shortcuts](shortcuts/index.md) and [Navigate DevTools with assisti
 You can dock DevTools in the right, left, or bottom of the browser, or undock it into a separate window.  See [Change DevTools placement (Undock, Dock to bottom, Dock to left)](customize/placement.md).
 
 
+<!-- ====================================================================== -->
+## Zooming DevTools or the rendered page
+
+The DevTools UI is implemented using HTML and CSS, and you can zoom it in and out like any other web page.  The zoom level for the DevTools portion of the browser is independent from the zoom level for the rendered webpage portion of the browser.
```
Comment (captainbrosset) [inline microsoft-edge/devtools-guide-chromium/overview.md:161]: I'd like to suggest a slight rephrasing to make the relationship between DevTools using HTML/CSS and the fact that you can zoom it more obvious:

"The DevTools UI is implemented using HTML and CSS, like web pages, which means you can zoom it in and out like any other web page."

File: microsoft-edge/devtools-guide-chromium/overview.md [hunk 2]
```diff
@@ -155,6 +155,30 @@ See [Keyboard shortcuts](shortcuts/index.md) and [Navigate DevTools with assisti
...
+
+There are 17 zoom levels: 8 zoom-out levels + 1 standard level (100%) + 8 zoom-in levels.  When you zoom the rendered page, the zoom percentage level is briefly displayed.
+
+
+To zoom the rendered page:
+
+1. If focus isn't already on the rendered page, click somewhere in the rendered page.
+1. Press `Ctrl`+`+` or `Ctrl`+`-` (Windows or Linux).  Or, press `Command`+`+` or `Command`+`-` (macOS).
+
+![Browser zoomed out, DevTools zoomed in.](overview-images/zoom-rendered-page.png)
```
Comment (captainbrosset) [inline microsoft-edge/devtools-guide-chromium/overview.md:171]: This documentation is about DevTools, so explaining how to zoom the rendered page doesn't make a lot of sense here I think. It's definitely ok to mention that both DevTools and the page can be zoomed with the same keyboard shortcuts, but I wouldn't necessarily add steps and a screenshot to explain how to do it in the page.

How about merging the 2 sub-sections like this:

```
To zoom the DevTools portion of the browser:

1. If focus isn't already on DevTools, click somewhere in DevTools.
1. Press `Ctrl`+`+` or `Ctrl`+`-` (Windows or Linux).  Or, press `Command`+`+` or `Command`+`-` (macOS).

![Browser zoomed in, DevTools zoomed out.](overview-images/zoom-devtools.png)

Alternatively, if you want to zoom the rendered page, click somewhere in the rendered page and use the same keyboard shortcuts.
```

**Review by captainbrosset — CHANGES_REQUESTED** [review]

File: microsoft-edge/devtools-guide-chromium/overview.md [hunk 3]
```diff
@@ -155,6 +155,30 @@ See [Keyboard shortcuts](shortcuts/index.md) and [Navigate DevTools with assisti
 You can dock DevTools in the right, left, or bottom of the browser, or undock it into a separate window.  See [Change DevTools placement (Undock, Dock to bottom, Dock to left)](customize/placement.md).
 
 
+<!-- ====================================================================== -->
+## Zooming DevTools or the rendered page
```
Comment (mikehoffms) [inline microsoft-edge/devtools-guide-chromium/overview.md:159]: @captainbrosset - There's no zoom in Settings it seems.  State that here?

Comment (captainbrosset) [reply microsoft-edge/devtools-guide-chromium/overview.md]: Oh you are right! I never realized that. This is a bug, I'll report it to the team. But in the meantime thanks for the docs about it.

**Pull request merged**